app/helpers/games_helper.rb

```diff
@@ -1,2 +1,17 @@
 module GamesHelper
-end
+	def color_block(row, col)
```

Comment:
set up your sublime to 2 spaces per tab and save as spaces rather than tabs.  Will make a cleaner looking git record.